automations/js/src/project_automation/prs.mjs

```
@@ -19,17 +19,15 @@ async function syncReviews(core, pr, prBoard, prCard) {
   core.debug(`PR review counts: ${reviewCounts}`)
   core.debug(`PR reviews decision: ${reviewDecision}`)
 
-  if (reviewDecision === 'APPROVED') {
-    core.info('Moving PR on the basis of review decision.')
+  if (reviewDecision === "APPROVED") {
+    core.info("Moving PR on the basis of review decision.")
     await prBoard.moveCard(prCard.id, prBoard.columns.Approved)
-  } else if (reviewDecision === 'CHANGES_REQUESTED') {
-    core.info('Moving PR on the basis of review decision.')
+  } else if (reviewDecision === "CHANGES_REQUESTED") {
+    core.info("Moving PR on the basis of review decision.")
     await prBoard.moveCard(prCard.id, prBoard.columns.ChangesRequested)
   } else if (reviewCounts.APPROVED === 1) {
```

This condition is unreachable, instead this should be the catch-all block.
```suggestion
  } else {
```